insights-notification-schemas-java/src/test/java/com/redhat/cloud/notifications/ingress/TestSerialization.java

```diff
@@ -92,6 +94,40 @@ void deserializeWithV1_0_0() {
         assertEquals("v1.0.0", deserializedAction.getVersion());
     }
 
+    @Test
+    void encodeAndDecodeV1_1_0() {
+        Registry registry = new Registry();
+        Encoder encoder = new Encoder();
+        Decoder decoder = new Decoder(registry);
```

Comment:
Not much important, but these three lines could be moved to a beforeEach method.

Reply:
I am not merging it so you can decide if you want to put the creation of these three objects in a before method or not :-)
if not, feel free to merge :)